web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -330,7 +407,7 @@ function notifyResult(operation, error) {
     const togBtn = document.getElementById('hide-but');
     triggerTransition(togBtn, transitionColor);
     if (error) {
-        console.error(`BmcSidePanel: ${operation} failed: ${error.message}`);
+        LOGS.log('E0040', { opname: operation, reason: error });
```

Comment:
An error became a log? Also, logs don't take error codes normally (even if they \*can\*).

Reply:
Good point.
- [ ] Use Localized String codes for logs